public/locales/en/dataset.json

```diff
@@ -22,8 +22,17 @@
     "uploadFiles": "Upload Files"
   },
   "alerts": {
-    "draftVersion": "<b>This draft version needs to be published</b>. When ready for sharing, please <b>publish</b> it so that others can see these changes",
-    "requestedVersionNotFound": "Info – Version {0} was not found. This is version {1}",
-    "unpublishedDataset": "Privately share this dataset before it is published: {0}"
+    "draftVersion": {
+      "heading": "This draft version needs to be published",
+      "alertText": "When ready for sharing, please <b>publish</b> it so that others can see these changes"
+    },
+    "requestedVersionNotFound": {
+      "heading": "Information",
+      "alertText": "Version {{requestedVersion}} was not found. This is version {{returnedVersion}}."
+    },
+    "unpublishedDataset": {
+      "heading": "Unpublished Dataset Private URL",
+      "alertText": "Privately share this dataset before it is published: {{privateUrl}}"
```

Comment:
This message "Privately share this dataset before it is published: {{privateUrl}}" is not being displayed in the SPA

JSF:
![image](https://github.com/IQSS/dataverse-frontend/assets/23359572/eddc79b5-8c74-40db-a1a7-0dc8f2ee897b)

SPA:
![image](https://github.com/IQSS/dataverse-frontend/assets/23359572/5afe642a-0b28-47ad-be17-f10308cbb32b)

Reply:
In the JSF, this message isn't displayed when the dataset is in Anonymized view, only when the dataset owner is in the normal view, after the privateUrlToken was created.  So I think we need to add a property to Dataset for the created token.  This would be a separate property from the privateUrlToken that is passed through searchParams.
I don't know if I'm explaining that clearly, so we can talk about it Monday. :)

Reply:
If I understood you well, we need js-dataverse to return a new Dataset property, the privateUrlToken, so it can be displayed in this alert from the normal view. Sounds good to me 👍